src/VotingMultipliers.sol

```diff
@@ -71,8 +80,8 @@ contract VotingMultipliers is Ownable2StepUpgradeable, IVotingMultipliers {
     /// @param _user The address of the user
     /// @param _multiplierIndexes Array of multiplier indexes to use
     /// @return The total multiplier value for the user
-    function getTotalMultipliers(address _user, uint256[] calldata _multiplierIndexes) public view returns (uint256) {
-        uint256 _totalMultiplier = 0;
+    function getTotalMultipliers(address _user, uint256[] calldata _multiplierIndexes) public returns (uint256) {
```

Comment:
get functions should have the view visibility. this won't be very useful on the frontend as is

Reply:
My issue with this change is mostly from a readability perspective. The function name is `getTotalMultipliers` but it is not a view function, which is inconsistent. The visibility should either be updated or the name of the function should change so that it's clear that state changes are being made when it's called.